src/main/java/com/seiama/javaducks/injection/OutdatedBannerInjection.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of javaducks, licensed under the MIT License.
+ *
+ * Copyright (c) 2023-2024 Seiama
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.seiama.javaducks.injection;
+
+import com.seiama.javaducks.configuration.properties.AppConfiguration;
+import com.seiama.javaducks.service.JavadocService;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Objects;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+@Component
+public class OutdatedBannerInjection implements Injection {
+
+  private final AppConfiguration configuration;
+  private final String template;
+
+  public OutdatedBannerInjection(final AppConfiguration configuration) {
+    this.configuration = configuration;
+    try {
+      this.template = Files.readString(Path.of(Objects.requireNonNull(getClass().getClassLoader().getResource("outdated-banner.html")).toURI()));
```

Comment:
Can we use a [`Resource`](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/core/io/Resource.html)/[`ClassPathResource`](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/core/io/ClassPathResource.html) instead?